Jenkinsfile

```diff
@@ -0,0 +1,24 @@
+// For ci.jenkins.io
+// https://github.com/jenkins-infra/documentation/blob/master/ci.adoc
+
+def mavenName = 'mvn'
+def jdkName = 'jdk8'
+
+node('highram&&docker') {
+    withEnv([
+        "JAVA_HOME=${tool jdkName}",
+        "PATH+MAVEN=${tool mavenName}/bin:${env.JAVA_HOME}/bin"
+    ]) {
+        changelog scm
+
+        sh 'docker build --build-arg=uid=$(id -u) --build-arg=gid=$(id -g) -t jenkins/ath src/main/resources/ath-container'
+        docker.image('jenkins/ath').inside {
```

Comment:
You need to add this:

`String containerArgs = '-v /var/run/docker.sock:/var/run/docker.sock -v $HOME/.m2:/var/maven/.m2'`

and change it to be:

`docker.image('jenkins/ath').inside(containerArgs) {`

Otherwise, `docker ps` will not be able to connect to the docker daemon and docker-based test will fail